src/test/scala/is/hail/methods/ExprSuite.scala

```diff
@@ -671,6 +675,21 @@ class ExprSuite extends SparkSuite {
     // FIXME: parser should accept minimum Long/Int literals
     // assert(eval[Long](Long.MinValue.toString+"L").contains(Long.MinValue))
     // assert(eval[Long](Long.MinValue.toString+"l").contains(Long.MinValue))
+    
+    assert(eval[Boolean]("let c = calls.noCall in c.isNotCalled() && !c.isHomVar() && !c.isHet() && !c.isHomRef()").contains(true))
```

Comment:
&& short circuits.  This hid the isHomRef bug.  You probably want these as separate tests.